Encode.xs

```diff
@@ -139,6 +140,31 @@ do_fallback_cb(pTHX_ UV ch, SV *fallback_cb)
 }
 
 static SV *
+do_bytes_fallback_cb(pTHX_ U8 *s, STRLEN slen, SV *fallback_cb)
+{
+    dSP;
+    int argc;
+    STRLEN i;
+    SV *retval = newSVpvn("",0);
```

Comment:
> Duplicated code is always a bad idea?

Agree, but in this case when it is needed to call `call_sv` with different argument count and types we cannot do nothing more... maybe just move those perl macros like ENTER or PUSHMARK into another macros, but that would lead to even complicated code.

> Defer the allocation until successful execution of callback?

Now when I'm looking at `do_fallback_cb` again, that function just returns new (non-mortal) SV* which is return value from perl function called via `call_sv`. So what is needed is to propagate mortal SV* in POPs from ENTER/LEAVE section for future processing. What is correct and safe way to do that? Calling `retval = newSVsv(POPs)` prior to FREETMPS?